dobromir-hristov/swift-docc-render
# Add sidenav Accessibility improvements

Bug/issue #, if applicable: 88194935

## Summary

This PR improves the AX on the sidebar

## Dependencies

https://github.com/apple/swift-docc-render/pull/51

## Testing

Make sure you download and unzip the .doccarchive - [SlothCreator.doccarchive.zip](https://github.com/apple/swift-docc-render/files/8082216/SlothCreator.doccarchive.1.zip)

Steps:

1. run VUE_APP_DEV_SERVER_PROXY=/path/to/SlothCreator.doccarchive npm run serve
2. Visit http://localhost:8080/documentation/slothcreator
3. Try using Tabbing or VO

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
